fuse/data/tokenizers/modular_tokenizer/inject_utils.py

```diff
@@ -130,6 +126,7 @@ def prepare_info_for_model_step(
         *,
```

Comment:
maybe build_scalars be a better name here?

Reply:
np, renamed